deps/build.jl

```diff
@@ -47,7 +47,15 @@ function find_libcuda()
             end
         end
     libcuda = Libdl.find_library(libcuda_name, libcuda_locations)
-    isempty(libcuda) && error("CUDA driver library cannot be found (specify the path to $(libcuda_name) using the CUDA_DRIVER environment variable).")
+    if isempty(libcuda)
+      warn("CUDA driver library cannot be found (specify the path to $(libcuda_name) using the CUDA_DRIVER environment variable).")
+      open(ext, "w") do fh
+          write(fh, """
+              ENV["CUDADRV_ONLY_LOAD"] = "true"
```

Comment:
I would prefer having a Julia variable being set instead of round tripping through the environment.